### PR TITLE
Fixes issue #550: Static attributes seem not supported in Configurati…

### PR DESCRIPTION
**Pull request by mgaggero**
…on Group provisioning

### CODE REVIEW

File: lib/services/devices/deviceService.js [hunk 1]
```diff
@@ -234,7 +234,7 @@ function mergeArrays(original, newArray) {
         newKeys = _.pluck(newArray, 'object_id'),
```
Comment (fgalan) [inline lib/services/devices/deviceService.js]: An entry in CHANGES_NEXT_RELEASE file should be added describing what this PR is fixing (have a look to existing content to CHANGES_NEXT_RELEASE entries to see what I'm referring to).

Comment (fgalan) [reply lib/services/devices/deviceService.js]: If this PR is solving more than one issue, then one entry per issue should be added.

**Comment by agaldemas** [comment]
It could be nice to integrate #544 proposal of modification in deviceService.js
since it fix the issue with Automatic device provisioning by means of Configuration Group
at least for dynamic attributes, 
for static_attributes the documentation (apiary) indicates it should be propagated to entity, which is not fulfilled.
I let you organize the PRs, but LGTM, thanks in advance

**Comment by agaldemas** [comment]
Dear all,
I've tested with the 2 fix proposed in this PR, it works OK with iotagent-ul and iotagent-json
static_attributes defined in group (service) are propagated to the entity
double LGTM ! ;O)

**Comment by fgalan** [comment]
Great to know @agaldemas ! :) Once the comment published above (https://github.com/telefonicaid/iotagent-node-lib/pull/551#discussion_r170318381) gets solver, PR will be merged.

**Comment by mgaggero** [comment]
Hi @agaldemas and @fgalan , I have aligned the PR with the master and populated the file CHANGES_NEXT_RELEASE with the description of the issues the PR fixes. Also the CI tests run successful.

**Comment by mgaggero** [comment]
@agaldemas @fgalan, thanks for the feedback and the independent tests :) !!!

File: lib/services/devices/deviceService.js (hunk 1 above)
Comment (fgalan) [reply lib/services/devices/deviceService.js]: Fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/551/commits/3753c201c702080240ab07947392e895ca7519c4

**Pull request merged**